tools/internal_ci/linux/grpc_e2e_performance_gke.cfg

```diff
@@ -16,7 +16,7 @@
 
 # Location of the continuous shell script in repository.
 build_file: "grpc/tools/internal_ci/linux/grpc_e2e_performance_gke.sh"
-timeout_mins: 120
+timeout_mins: 240
```

Comment:
why would we need a timeout so high (when we know the avg duration is only 90mins  and we want to keep it that way?). Increasing to e.g. 180m to allow for some buffer seems fair, but 240m seems excessive.